packages/search-ui/src/SearchDriver.ts

```diff
@@ -124,7 +162,7 @@ export default class SearchDriver {
       console.warn(
         "Search UI Debugging is enabled. This should be turned off in production deployments."
       );
-      window.searchUI = this;
+      if (typeof window !== "undefined") window["searchUI"] = this;
```

Comment:
so what needs to happen is give typescript information on what window is and it's API so that we can add a reference to SearchUI. I've been lazy here (the better way would to declare what window module is) but with the check, im allowed to add a property to the variable without telling typescript that window is a module and can have an optional property of SearchUI.

Reply:
Ah, nice trick, good to know!